config/routes.rb

```diff
@@ -77,7 +77,7 @@ def not_implemented
             constraints: ToolMountPoint.for(:advice_plans)
 
       mount DecisionTrees::Engine => '/tools/:tool_id',
-            constraints: ToolMountPoint.for(:decision_trees)
+            constraints: { tool_id: %r{health-check-questions|workplace-pension-advice-tool} }
```

Comment:
My point is should `%r{health-check-questions|workplace-pension-advice-tool}` belong in the routes file or in the tool mount point.